packages/py3_flit_core.rb

```diff
@@ -0,0 +1,33 @@
+require 'package'
+
+class Py3_flit_core < Package
+  description 'Flit provides simplified packaging of Python modules—core portions.'
+  homepage 'https://flit.pypa.io/'
+  @_ver = '3.7.1'
+  version @_ver
+  license 'MIT'
+  compatibility 'all'
+  source_url 'https://github.com/pypa/flit.git'
+  git_hashtag @_ver
+
+  binary_url({
+    i686: 'file:///usr/local/tmp/packages/py3_flit_core-3.7.1-chromeos-i686.tar.zst',
+  x86_64: 'file:///usr/local/tmp/packages/py3_flit_core-3.7.1-chromeos-x86_64.tar.zst'
```

Comment:
Local files?

Reply:
oops, that needs to be fixed, i'll do it when i boot up my PC tomorrow.